.packit.yaml

```diff
@@ -19,62 +19,105 @@ jobs:
   - centos-stream-8-x86_64
   - centos-stream-9-x86_64
 
-- &test
+- <<: *build
+  trigger: commit
+  branch: "gh-readonly-queue/.*"
+
+- &test-static-checks
   job: tests
   trigger: pull_request
   fmf_path: tests/tmt-plans
+  identifier: /static-checks
+  tmt_plan: /static-checks
   targets:
     epel-7:
       distros: [ centos-7 ]
     centos-stream-8: {}
     centos-stream-9: {}
-  identifier: /static-checks
-  tmt_plan: /static-checks
 
 # when modifying this, modify also tests/tmt-plans/
-- <<: *test
+
+- &test-hardening-ansible-anssi_bp28_high
+  <<: *test-static-checks
```

Comment:
Could you please explain this? (or link me documentation where it is described)
Commit message doesn't say much and the change from
```
- <<: *test
```
to
```
- &test-hardening-ansible-anssi_bp28_high
  <<: *test-static-checks
```
doesn't give hint either.

I don't see `test-hardening-ansible-anssi_bp28_high` anywhere. Both, GH Action jobs and Testing Farm, display identifier (or maybe tmt plan name?) `/hardening/host-os/ansible/anssi_bp28_high`. That's why I don't get `&test-hardening-ansible-anssi_bp28_high`. And regarding `<<: *test-static-checks`, no idea at all.

Reply:
I re-used existing syntax that was there for merge queues. It's all using https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

Reply:
Got it. Mainly the `test-static-checks` confused me. Would it be possible/make sense to have it like a template without a trigger?
```
- &tests
  job: tests
  fmf_path: tests/tmt-plans
  targets:
    epel-7:
      distros: [ centos-7 ]
    centos-stream-8: {}
    centos-stream-9: {}
```
and then refer to that everywhere?
```
- &test-static-checks
  <<: *tests
  trigger: pull_request
  identifier: /static-checks
  tmt_plan: /static-checks

- &test-hardening-ansible-anssi_bp28_high
  <<: *tests
  trigger: pull_request
  identifier: /hardening/host-os/ansible/anssi_bp28_high
  tmt_plan: /hardening/host-os/ansible/anssi_bp28_high
  targets:
    centos-stream-8: {}
    centos-stream-9: {}
```

Reply:
Unfortunately, no. YAML doesn't allow anchors that wouldn't also serve as valid data. There is no equivalent to TMT like definition of metadata, where the presence of `/child/entries` render any parent-defined metadata inert, to be used only for inheritance.

If YAML would support that, it would greatly simplify everything and we wouldn't need to duplicate so much text.